docker/cygnus-ngsi/agent.conf

```diff
@@ -221,6 +221,21 @@ cygnus-ngsi.sinks.elasticsearch-sink.ssl = false
 #cygnus-ngsi.sinks.elasticsearch-sink.cache_flash_interval_sec = 0
 
 
+# NGSIArcGisSink configuration
+cygnus-ngsi.sinks.arcgis-sink.type = com.telefonica.iot.cygnus.sinks.NGSIArcGisSink
+cygnus-ngsi.sinks.arcgis-sink.channel = arcgis-channel-santander
```

Comment:
arcgis-channel-santander -> arcgis-channel ?

Reply:
Yes, I changed it